config/routes.rb

```diff
@@ -40,4 +41,9 @@
     put 'profile/:id' => 'profile#update', as: :profile
     post 'invite_friends' => 'profile#invite_friends', as: :invite_friends
   end
+
+  root to: redirect("/#{I18n.locale}", status: 302), as: :redirected_root
+  get "/*path", to: redirect("/#{I18n.locale}/%{path}", status: 302),
+      constraints: {path: /(?!(#{I18n.available_locales.join("|")})\/).*/},
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Space inside { missing.<br>Space inside } missing.